### ISSUE TITLE
Broken link in README.md

### ISSUE THREAD

**Issue by binarymist** [paragonie/awesome-appsec#32]
https://github.com/paragonie/awesome-appsec#salted-password-hashing-doing-it-right-2014
https://github.com/paragonie/awesome-appsec#salted-password-hashing---doing-it-right-2014

Let me know if you are accepting PRs?

**Comment by paragonie-scott** [comment]
Yeah, we need to fix the URL generator, because it's generated from JSON data files.

I wish Github open sourced their algorithm for those, so we could guarantee consistency.

**Comment by binarymist** [comment]
Ah, I see, I think I've come across something similar when I started writing my book on github, yeah, pain.

**Comment by sstok** [comment]
You can use `### <a id="unique-anchor-name">Title</a>` to have permanent anchor links.

And GitHub has actually open-sourced there Markup renderer 👍 
https://github.com/github/markup

**Comment by binarymist** [comment]
Looks like the issue is the ` - ` (hyphen with space on either side)

**Comment by 4chm3d** [comment]
Hi! What about replacing the "-" in name field? #54
